here-naksha-app-service/src/test/java/com/here/naksha/app/service/PropSearchTestUtil.java

```diff
@@ -0,0 +1,86 @@
+package com.here.naksha.app.service;
+
+import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.util.stream.Stream;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.here.naksha.app.common.TestUtil.urlEncoded;
+
+public class PropSearchTestUtil {
+
+  private static Stream<Arguments> queryParams() {
```

Comment:
At first I thought it was not used because of IntelliJ warnings ;)  When you use `@MethodSource` that references method from the same scope (ie private method in the same class) then IJ is able to connect the dots and it does not give this warning.

This is minor, but maybe it's worth to somehow indicate usage of this (like simple comment or dummy proxy method in test class?). It's nothing more than minor remark though, can be left as is

Reply:
Done.
Fixed with proxy method.